game/scripts/vscripts/Sections/Chapter3/Chapter3.ts

```diff
@@ -287,9 +289,33 @@ const onStart = (complete: () => void) => {
             // Dialog
             tg.audioDialog(LocalizationKey.Script_3_Opening_1, LocalizationKey.Script_3_Opening_1, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
             tg.audioDialog(LocalizationKey.Script_3_Opening_2, LocalizationKey.Script_3_Opening_2, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
-            tg.audioDialog(LocalizationKey.Script_3_Opening_3, LocalizationKey.Script_3_Opening_3, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
-            tg.audioDialog(LocalizationKey.Script_3_Opening_4, LocalizationKey.Script_3_Opening_4, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
-            tg.audioDialog(LocalizationKey.Script_3_Opening_5, LocalizationKey.Script_3_Opening_5, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+            tg.audioDialog(LocalizationKey.Script_3_Opening_3, LocalizationKey.Script_3_Opening_3, ctx => ctx[CustomNpcKeys.SlacksMudGolem]),
+            tg.forkAny([
+                tg.audioDialog(LocalizationKey.Script_3_Opening_5, LocalizationKey.Script_3_Opening_5, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+                tg.seq([
```

Comment:
All the camera pans felt very very quick to me. I couldn't understand what I'm seeing. Can you please make it slightly slower? Maybe with a very slight wait time between each camp.

Also, to make it similar to the section where you see the wards, can we make it so it doesn't complete when the audio's skipped? In the wards section, you can skip the dialog, but you're still watching the ward locations. Only continue to the next dialog when the camera finishes showing all the locations and pans back to the player's hero.